pkg/kubelet/stats/cri_stats_provider.go

```diff
@@ -206,6 +206,7 @@ func (p *criStatsProvider) listPodStatsPartiallyFromCRI(ctx context.Context, upd
 		cs := p.makeContainerStats(stats, container, rootFsInfo, fsIDtoInfo, podSandbox.GetMetadata(), updateCPUNanoCoreUsage)
 		p.addPodNetworkStats(ps, podSandboxID, caInfos, cs, containerNetworkStats[podSandboxID])
 		p.addPodCPUMemoryStats(ps, types.UID(podSandbox.Metadata.Uid), allInfos, cs)
+		p.addSwapStats(ps, types.UID(podSandbox.Metadata.Uid), allInfos, cs)
```

Comment:
can you also extend the PodSandboxStats request in `staging/src/k8s.io/cri-api/pkg/apis/runtime/v1/api.proto` and then adapt `listPodStatsStrictlyFromCRI` to add them as well? It will fail for now but CRI needs to be taught how to gather swap stats too